main.py

```diff
@@ -1,10 +1,40 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_info = json.load(f)
+
+    for pl in players_info.keys():
+        race, created = Race.objects.get_or_create(
+            name=players_info[pl]["race"]["name"],
+            description=players_info[pl]["race"]["description"]
+        )
+
+        if created and players_info[pl]["race"]["skills"]:
+            for skill in players_info[pl]["race"]["skills"]:
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+
+        guild = None
+        if players_info[pl]["guild"]:
```

Comment:
Ensure that `players_info[pl]["guild"]` is not `None` or an empty dictionary to prevent errors when accessing its keys.